docker-files/zeeguu-web/Dockerfile

```diff
@@ -0,0 +1,104 @@
+FROM python:3.6
+
+# Build arguments with default values, overwrite at build-time using
+# "--build-arg <varname>=<value>"
+ARG DEFAULT_ZEEGUU_API="https://zeeguu.unibe.ch/api"
+# The public IP where the zeeguu API can be reached on
+# It will be used in the static JS
+ARG ZEEGUU_API_PUBLIC=http://127.0.0.1:9001
+ARG ZEEGUU_API=http://127.0.0.1:9001
```

Comment:
can we improve the names?
ZEEGUU_API__EXTERNAL #accessed from the client code, might be different than the internal because of apache might mask the port
ZEEGUU_API__INTERNAL #accessed from python